src/scss/components/_btn.scss

```diff
@@ -5,9 +5,11 @@
   // Reset default <a> styles
   text-decoration: none;
 
+  font-weight: 800;
```

Comment:
since we use `font-weight: 800` quite a few times here, lets create a variable for it like `font-weight: $bolder`